fluent/src/bundle.js

```diff
@@ -64,15 +64,6 @@ export default class FluentBundle {
     this._intls = new WeakMap();
   }
 
-  /*
-   * Return an iterator over public `[id, message]` pairs.
-   *
-   * @returns {Iterator}
-   */
-  get messages() {
-    return this._messages[Symbol.iterator]();
-  }
-
```

Comment:
It would be good to retain some way of iterating over a bundle's messages. Atm. e.g. `print()` in `tools/format.js` ends up needing to access `bundle._messages`.

How about something like this?

```js
get messageKeys() {
  return this._messages.keys();
}
```

Reply:
I don't think we need to add this right now. I don't know of good runtime use-cases for iterating over all messages in a bundle. I usually prefer to keep the API surface as small as possible, and extend it only when needed.

The code in `tools/format.js` should use the result of the full-AST parsing from `fluent-syntax` to build an iterable of message keys for the purpose of printing all messages. I intend to fix it in a follow-up but I can also do it here.